orchestration/dagster_orchestration/hca_manage/main.py

```diff
@@ -117,7 +119,7 @@ def check_data(args, host, parser):
         hca.check_for_all()
 
 
-def create_snapshot(args, host):
+def create_snapshot(args: argparse.Namespace, host: str) -> JobId:
```

Comment:
the repetition of this argument signature makes me think that these methods should be wrapped in a class that accepts the args in its initializer, but that's out of scope for this PR. small bite?

Reply:
I think we need to refactor this script into separate scripts that are more targeted, it's become a bit unwieldy to use.